napari/layers/labels/labels.py

```diff
@@ -802,30 +802,7 @@ def _on_editable_changed(self) -> None:
             self.mode = Mode.PAN_ZOOM
             self._reset_history()
 
-    def _lookup_with_low_discrepancy_image(self, im, selected_label=None):
-        """Returns display version of im using low_discrepancy_image.
-
-        Passes the image through low_discrepancy_image, only coloring
-        selected_label if it's not None.
-
-        Parameters
-        ----------
-        im : array or int
-            Raw integer input image.
-        selected_label : int, optional
-            Value of selected label to color, by default None
-        """
-        if selected_label:
-            image = np.where(
-                im == selected_label,
-                low_discrepancy_image(selected_label, self._seed),
-                0,
-            )
-        else:
-            image = np.where(im != 0, low_discrepancy_image(im, self._seed), 0)
-        return image
-
-    def _as_type(self, data, selected_label=None):
+    def _as_type(self, data):
```

Comment:
When I work with #6179 it is confusing for me why use such a simple method. I think that such a simple docstring may help understand why it is created. What did you think? 

```suggestion
    def _as_type(self, data: np.ndarray) -> np.ndarray[np.float32]:
        """single function to synchronize type conversion"""
```

Reply:
How about I rename the function to `_to_texture_dtype`? Honestly it was quickly named in a hacking session in the middle of #3308, replacing `_raw_to_displayed` which was no longer needed.

Reply:
good idea. The name is more descriptive. maybe even `_to_vispy_texture_dtype`?

Reply:
Done!